config/example/admin_ranks.txt

```diff
@@ -1,38 +0,0 @@
-########################################################################################
```

Comment:
why did you delete the admin ranks file

Reply:
yeah that one was something I've been trying to figure out. so it seems to me when I did the usual testing for admin permissions it added itself onto the PR?
I think I'mma just redo the PR itself.

Reply:
Well unless there is a way to remove that. because the admin files were completely unintentional and a fluke. I just can't figure out how to undo it from here without redoing the PR.

Reply:
but yeah that is completely a bug.